### PR TITLE
MAINT-2144 Import/Export Identifier file

### PR DESCRIPTION
**Pull request by QuyenLy87**

### CODE REVIEW

File: src/main/java/org/snomed/snowstorm/core/rf2/rf2import/ImportService.java [hunk 1]
```diff
@@ -195,7 +198,7 @@ private Integer fullImport(final InputStream releaseFileStream, final String bra
 
 		final FullImportComponentFactoryImpl importComponentFactory = getFullImportComponentFactory(branchPath);
 		try {
-			releaseImporter.loadFullReleaseFiles(releaseFileStream, loadingProfile, importComponentFactory);
+			releaseImporter.loadFullReleaseFiles(releaseFileStream, loadingProfile, importComponentFactory, true);
```
Comment (CoderMChu) [inline src/main/java/org/snomed/snowstorm/core/rf2/rf2import/ImportService.java:201]: Multi threaded when loading looks great.

**Review by CoderMChu — COMMENTED** [review]
@QuyenLy87 It looks good only with a couple of minor comments. Thanks.

**Comment by QuyenLy87** [comment]
Thanks @CoderMChu . I've updated regarding your suggestions. Please have a look. Thanks

**Review by CoderMChu — APPROVED** [review]
@QuyenLy87 It looks good. Thanks.

**Comment by QuyenLy87** [comment]
The changes were merged back into develop using cherry-picking.

**Pull request closed without merging**